pilot/pkg/config/kube/gateway/backend_policies.go

```diff
@@ -147,6 +151,14 @@ func DestinationRuleCollection(
 					lbSet = true
 					spec.TrafficPolicy.LoadBalancer = pol.LoadBalancer
 				}
+				if pol.RetryBudget != nil {
```

Comment:
This is interesting. Obviously we allow BackendTLS and BackendTrafficPolicy together.

But should we merge sessionPersistence and RetryBudget across 2 BackendTrafficPolicy or only take 1?

Reply:
I suspect we only want to take a single BackendTrafficPolicy but we should double check the spec.